sdk/react/src/shared/BottomOverlay/MarkersContainer/Marker/styles/commonMarker.style.js

```diff
@@ -2,16 +2,69 @@
 
 import { MARKERS_SIZES } from '../../../../../constants';
 
-export default {
+export const createRootStyle = () => ({
   backgroundColor: 'black',
   borderColor: '#4E4E4E',
-  borderRadius: 2,
+  borderRadius: MARKERS_SIZES.BORDER_RADIUS,
   borderWidth: MARKERS_SIZES.BORDER_WIDTH,
   bottom: MARKERS_SIZES.DISTANCE_FROM_BOTTOM,
   padding: MARKERS_SIZES.PADDING,
   position: 'absolute',
   shadowColor: 'black',
-  shadowOffset: { height: 2, width: 0 },
+  shadowOffset: {
+    height: 2,
+    width: 0,
+  },
   shadowOpacity: 0.5,
   shadowRadius: 4,
+});
+
+export const createTriangleStyle = () => ({
+  // Create triangle shape using border magic.
+  borderColor: 'transparent',
+  borderTopColor: '#4E4E4E',
+  borderWidth: MARKERS_SIZES.DISTANCE_FROM_BOTTOM,
+  borderBottomWidth: 0,
+  bottom: -1 * MARKERS_SIZES.DISTANCE_FROM_BOTTOM,
+  height: 0,
+  position: 'absolute',
+  width: 0,
+});
+
+export const restrainLeftPositionWithinContainer = (width: number, leftPosition: number, containerWidth: number) => {
```

Comment:
I suggest covering these helpers with tests for better safety

Reply:
@ooyala-amohnatkin please revisit for test